app/src/main/java/com/isc/hermes/generators/GeneratorManager.java

```diff
@@ -0,0 +1,67 @@
+package com.isc.hermes.generators;
+
+import com.isc.hermes.model.incidents.*;
+import org.bson.types.ObjectId;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * This class handles incident generators to randomly generate incidents.
+ */
+public class GeneratorManager {
```

Comment:
The name of this class is ambiguous, a suggestion would be "IncidentGeneratorManager"